buildSrc/src/main/java/com/uber/okbuck/core/util/FileUtil.java

```diff
@@ -38,15 +38,12 @@ public static String getRelativePath(File root, File f) {
 
     @SuppressWarnings("ResultOfMethodCallIgnored")
     public static void copyResourceToProject(String resource, File destination) {
-        try {
-            InputStream inputStream = FileUtil.class.getResourceAsStream(resource);
-            destination.getParentFile().mkdirs();
-            OutputStream outputStream = new FileOutputStream(destination);
+        destination.getParentFile().mkdirs();
```

Comment:
Actually, it can't. This was why this PR was originally failing on ci. The next line, `new FileOutputStream(destination)` depends on this file having been created.
Luckily, this method doesn't throw an IOException.